native/dll/Win32DLL/SageTVWin32DLL.cpp

```diff
@@ -202,23 +202,24 @@ JNIEXPORT jboolean JNICALL Java_sage_Sage_writeStringValue(JNIEnv *env,
 	const char* valueNameString = env->GetStringUTFChars(valueName, 0);
 	const char* valueString = env->GetStringUTFChars(value, 0);
 	HKEY rootKey;
+	rootKey = NULL;  // initialize to resolve C6001
 	HKEY myKey;
 	jboolean rv = JNI_FALSE;
 	if (root == sage_Sage_HKEY_CLASSES_ROOT) rootKey = HKEY_CLASSES_ROOT;
 	else if (root == sage_Sage_HKEY_CURRENT_CONFIG) rootKey = HKEY_CURRENT_CONFIG;
 	else if (root == sage_Sage_HKEY_CURRENT_USER) rootKey = HKEY_CURRENT_USER;
 	else if (root == sage_Sage_HKEY_LOCAL_MACHINE) rootKey = HKEY_LOCAL_MACHINE;
 	else if (root == sage_Sage_HKEY_USERS) rootKey = HKEY_USERS;
-
-	if (rootKey && RegCreateKeyEx(rootKey, keyString, 0, 0, REG_OPTION_NON_VOLATILE, KEY_ALL_ACCESS,
-		0, &myKey, 0) == ERROR_SUCCESS)
+	if (rootKey != NULL)

```

Comment:
You don't need this line because rootKey is checked in the conditional 2 lines down.

Reply:
Done  + the other one too :)